core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/logging/LogManager.scala

```diff
@@ -44,3 +44,45 @@ abstract class LogManager {
   def terminate(): Unit
 
 }
+
+class EmptyLogManagerImpl(
+    networkCommunicationActor: NetworkCommunicationActor.NetworkSenderActorRef
+) extends LogManager {
```

Comment:
change `LogManager` to trait from abstract class

Reply:
done